_posts/2017/2017-11-12-use-a-pull-request-template.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Use a pull request template'
+description: 'It helps you answer what, why & how.'
+layout: article
+category: ruby
+image:
+  base: '2017/use-a-pull-request-template'
+  alt: 'Man pulling rope'
+  credit: 'Stijn Swinnen'
+  source: 'https://unsplash.com/photos/Q8FHN3qSq2w'
+---
+
+If you're submitting work for review, pull requests are a great way to group related changes and discuss approaches and improvements. The audience for a pull request can be your immediate co-workers, other interested parties or even your future self.
+
+It should describe the ‘what’, ‘why’ and high-level ‘how’ of the changes in your branch. It is important the description is detailed and sets the context for the changes you have made .
+
+Github offers the ability to define pull request templates. See [GitHub’s documentation on pull request templates](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/).
```

Comment:
'Github' to 'GitHub'.

Reply:
Maybe change the link text to 'their documentation on this', rather than repeating 'GitHub' and then 'pull request templates' at the end of two sentences in a row.